src/sentry/static/sentry/app/views/organizationIntegrations/index.jsx

```diff
@@ -79,10 +145,50 @@ export default class OrganizationIntegrations extends AsyncComponent {
             <Box px={2} flex="1">
               {t('Integrations')}
             </Box>
+            {this.state.reloading && <StyledLoadingIndicator mini />}
           </PanelHeader>
           <PanelBody>{integrations}</PanelBody>
         </Panel>
       </React.Fragment>
     );
   }
 }
+
+const StyledLoadingIndicator = styled(LoadingIndicator)`
+  position: absolute;
+  right: 7px;
+  top: 50%;
+  transform: translateY(-16px);
```

Comment:
👀

Reply:
Pondered on this one for a hour or so with @Chrissy

Reply:
Yeah unfortunately the parent doesn’t inherit the true height of the svg—this is a problem I’ve encountered other places and want to get to the bottom of eventually.